### PR TITLE
REGION: add support for vwo data region

### PR DESCRIPTION
**Pull request by vwo-rohittyagi**
_Added support to choose you data region  which provided by VWO

### CODE REVIEW

File: packages/destination-actions/src/destinations/vwo/generated-types.ts [hunk 1]
```diff
@@ -9,4 +9,8 @@ export interface Settings {
    * VWO Fullstack SDK Key. It is mandatory when using the VWO Fullstack suite.
    */
   apikey?: string
+  /**
+   * VWO Region to sync data to. Default is US
+   */
+  region: string
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/vwo/generated-types.ts:15]: Hi @vwo-rohittyagi - adding a new 'required' field will break data collection for existing users. The reason for this is that the 'region' field will resolve to `undefined` for customers who already have the Integration set up.  

The way around this is to do the following: 

1. Make region an optional field. 
2. In the perform() function, check if settings.region === undefined, and if it is then send data to the default region.

File: packages/destination-actions/src/destinations/vwo/trackEvent/index.ts [hunk 2]
```diff
@@ -83,7 +83,8 @@ const action: ActionDefinition<Settings, Payload> = {
       settings.vwoAccountId
     )
     structuredPayload.d.event.props.vwoMeta['ogName'] = payload.name
-    const endpoint = `https://dev.visualwebsiteoptimizer.com/events/t?en=${sanitisedEventName}&a=${settings.vwoAccountId}`
+    const host = hosts[settings.region] || 'https://dev.visualwebsiteoptimizer.com'
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/vwo/trackEvent/index.ts:86]: nit

```suggestion
    const host = hosts[settings.region] || hosts.US
```

**Comment by joe-ayoub-segment** [comment]
hi @vwo-rohittyagi thanks for the PR. 
I reviewed and left a request for a change. 
Also the CI checks are failing. Could you look at the tests please?
Best regards,
Joe

File: packages/destination-actions/src/destinations/vwo/generated-types.ts (hunk 1 above)
Comment (vwo-rohittyagi) [reply packages/destination-actions/src/destinations/vwo/generated-types.ts]: Hi @joe-ayoub-segment ,
Thank you for reviewing the PR. I have implemented the suggested changes. Could you please review it again and prioritize its deployment? Some of our customers are currently blocked by this feature.
Thanks,
Rohit

**Comment by vwo-rohittyagi** [comment]
Hi @joe-ayoub-segment ,
We have  make required changes and also fixed test case. Please review
Thanks,
Rohit

**Review by artistictechie — APPROVED** [review]

**Review by joe-ayoub-segment — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @vwo-rohittyagi PR deployed. Please confirm the change looks good.

**Pull request merged**